grpc-gcp/src/main/java/com/google/grpc/gcp/GcpManagedChannel.java

```diff
@@ -76,10 +79,16 @@
   @GuardedBy("bindLock")
   final Map<String, ChannelRef> affinityKeyToChannelRef = new HashMap<String, ChannelRef>();
 
+  // Map from a broken channel id to the remapped affinity keys (key => ready channel id).
+  private final ConcurrentHashMap<Integer, ConcurrentHashMap<String, Integer>> fallbackMap =
+      new ConcurrentHashMap<>();
+
   @VisibleForTesting
   @GuardedBy("this")
   final List<ChannelRef> channelRefs = new ArrayList<ChannelRef>();
 
+  private final HashMap<Integer, ChannelRef> channelRefById = new HashMap<>();
```

Comment:
IMO, convention is to declare it as Map and assign Hashmap to it. basically, left side should be Map (and right side is, as you already have it, HashMap)

Reply:
Oops. Thanks! Done.